### PR TITLE
bats,{go/cmd/dolt/commands}: Hide doc schema for schema show w/no table args

### PR DESCRIPTION
**Pull request by katiemcculloch**

### CODE REVIEW

File: bats/docs.bats [hunk 1]
```diff
@@ -618,6 +618,15 @@ SQL
     run dolt schema show dolt_docs
     [ "$status" -eq 1 ]
     [[ "$output" =~ "'dolt_docs' is not a valid table name" ]] || false
+    run dolt schema show
+    [ "$status" -eq 1 ]
+    [[ "$output" =~ "usage: dolt schema show" ]] || false
```
Comment (timsehn) [inline bats/docs.bats]: Don't think this regex is necessary. When there are dolt_docs and no other table dolt schema show should error.

File: bats/docs.bats [hunk 2]
```diff
@@ -618,6 +618,15 @@ SQL
     run dolt schema show dolt_docs
     [ "$status" -eq 1 ]
     [[ "$output" =~ "'dolt_docs' is not a valid table name" ]] || false
+    run dolt schema show
+    [ "$status" -eq 1 ]
```
Comment (zachmu) [inline bats/docs.bats]: This doesn't make sense to me. Why should this command fail?

File: bats/docs.bats (hunk 1 above)
Comment (timsehn) [reply bats/docs.bats]: No need to test the help message.

File: bats/docs.bats (hunk 2 above)
Comment (timsehn) [reply bats/docs.bats]: Basically if only dolt_docs in the schema, error as if there is no schema...right Katie?

Comment (katiemcculloch) [reply bats/docs.bats]: @zachmu `dolt schema show` should fail and return the print the command usage if there are no tables on the working root (this was also the logic before I started tweaking it for docs).

We don't want to expose `dolt_docs`, and if it is the only table on the working root, we want the behavior to be as if there are no tables on the working root.

File: bats/docs.bats (hunk 1 above)
Comment (katiemcculloch) [reply bats/docs.bats]: thanks, removed the message assertion

File: bats/docs.bats (hunk 2 above)
Comment (zachmu) [reply bats/docs.bats]: I think this is actually a bug. No tables should not be an error. An empty schema is a valid repo state. It should behave the same as dolt ls when there are no tables: print "No tables in working set" and exit 0.

Everything else you have here looks fine. You can address the no tables bug in a separate PR if you want.

Comment (katiemcculloch) [reply bats/docs.bats]: Just pushed a commit that includes these updates: https://github.com/liquidata-inc/dolt/pull/337/files#diff-8dfc32e83b91e8c830d4d8bd789de412R132

**Pull request merged**